exercise_09/getDistanceMessageFromSumTo100.js

```diff
@@ -0,0 +1,56 @@
+/** 
+ * Write JS function "getDistanceMessageFromSumTo100" to generate a random number (in between 0-100) and sum it to given number
+  - if sum result exceeds 100, then return message "Sum with value N exceeds in N from number 100"
+  - otherwise, return message "Sum with value N is left in N from number 100"
+ * input: number (value) 
+ * output: string
+ */
+
+/**
+ * CASE #1
+ * input: 70
+ * random (generated in function): 40
+ * output: "Sum with value 110 exceeds in 10 from number 100"
+ */
+
+/**
+ * CASE #2
+ * input: 30
+ * random (generated in function): 40
+ * output: "Sum with value 70 isd left in 30 from number 100"
+ */
+
+import generateRandomNumberInRange from "../exercise_03/generateRandomNumberInRange.js";
+
+/**
+ * Generates a random number between 0 and 100 and sume it to given number
+ * @param {number} value Given number to sum to random number in range from 0 - 100
+ * @returns {string}
+ */
+const getDistanceMessageFromSumTo100 = function (value) {
+  const resultSumNumber = Math.round(
```

Comment:
One line, one thing

```suggestion
 const randomNumber = generateRandomNumberInRange(0, 100);
  const result = value + randomNumber
```

FIX: needs work